plugins/tableselection/plugin.js

```diff
@@ -269,15 +270,19 @@
 		if ( !evt.data.getTarget().getName ) {
 			return;
 		}
+		// Prevent of applying table selection when widget is selected.
+		// Mouseup remain possibility to finish table selection when user release mouse button above widget in table.
+		if ( evt.name !== 'mouseup' && CKEDITOR.plugins.widget && CKEDITOR.plugins.widget.isDomWidget( evt.data.getTarget() ) ) {
```

Comment:
This line introduced one unexpected (I think), behaviour:

Before:
![nov-03-2017 14-11-53](https://user-images.githubusercontent.com/1061942/32375230-262cbfac-c0a1-11e7-85b8-4f52ac94c2fd.gif)

After:
![nov-03-2017 14-11-35](https://user-images.githubusercontent.com/1061942/32375241-327df686-c0a1-11e7-995e-e1d644ba1459.gif)

so when dragging mouse over widgets, cells are not selected with fake selection as they would normally be.

Reply:
Extracted to #1125